src/store/createStore.ts

```diff
@@ -322,10 +330,23 @@ export const createStore = (options: CreateStoreOptions) =>
             set((state) => {
               state.transaction.sign.signedTx = tx;
             }),
+          updateBipPath: (bipPath: string) =>
+            set((state) => {
+              state.transaction.sign.bipPath = bipPath;
```

Comment:
We are saving the BIP path in the store, but it does not persist when the page is refreshed. Should we persist it if it's already in the store?